src/main/java/nextstep/session/domain/EnrolledStudents.java

```diff
@@ -1,19 +1,41 @@
 package nextstep.session.domain;
 
+import nextstep.payments.domain.PaymentPolicy;
 import nextstep.users.domain.NsUser;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class EnrolledStudents {
 
-    List<NsUser> students = new ArrayList<>();
+    private Long sessionId;
 
-    public void add(NsUser user) {
-        students.add(user);
+    public EnrolledStudents() {
```

Comment:
```suggestion
        this(0L, new ArrayList<Long>());
```
위와 같이 부생성자가 주생성자를 호출하도록 구현하는 것은 어떨까?